app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -71,6 +71,62 @@ def translate_git_tag(inputs)
         def translate_script(inputs)
           inputs['content']
         end
+
+        def translate_github_status(inputs)
+          return '# Invalid github-status step configuration!' unless required_github_status_inputs(inputs)
+
+          params = extract_github_status_params(inputs)
+          generate_github_status_script(params)
+        end
+
+        def extract_github_status_params(inputs)
+          {
+            auth_token: inputs['auth_token'],
+            api_base_url: inputs['api_base_url'].to_s.chomp('/'),
+            repo_path_command: extract_repo_path(inputs['repository_url']),
+            commit_hash: inputs['commit_hash'],
+            state_command: determine_status(inputs['set_specific_status']),
+            target_url: inputs['pipeline_build_url'] || inputs['build_url'] || '',
+            description: inputs['description'] || '',
+            context: inputs['context'] || 'default'
+          }
+        end
```

Comment:
if you are just trying to return a dictionary with good defaults without changing their names:provide good defaults a better way would be to have a hash with the default values and then just call `merge` with the inputs:

```suggestion
          defaults = {
            KEYS: VALUES...
          }
          
          generate_github_status_script(defaults.merge(inputs))
        end
```